features/datum.feature

```diff
@@ -75,150 +71,157 @@ Functionaliteit: leveren van een datum
-      | europeesKiesrecht     | einddatumUitsluiting      | JaarDatum      | 20300000 |            | 2030 |       |          | 2030         |
-      | uitsluitingKiesrecht  | einddatum                 | JaarDatum      | 20300000 |            | 2030 |       |          | 2030         |
       | geboorte              | datum                     | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | overlijden            | datum                     | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | opschortingBijhouding | datum                     | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | verblijfstitel        | datumIngang               | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | verblijfstitel        | datumEinde                | JaarMaandDatum | 20300700 |            | 2030 | 7     |          | juli 2030    |
-      | europeesKiesrecht     | einddatumUitsluiting      | JaarMaandDatum | 20300700 |            | 2030 | 7     |          | juli 2030    |
-      | uitsluitingKiesrecht  | einddatum                 | JaarMaandDatum | 20300700 |            | 2030 | 7     |          | juli 2030    |
-      | immigratie            | datumVestigingInNederland | Datum          | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | immigratie            | datumVestigingInNederland | DatumOnbekend  | 00000000 |            |      |       | true     | onbekend     |
-      | immigratie            | datumVestigingInNederland | JaarDatum      | 20200000 |            | 2020 |       |          | 2020         |
-      | immigratie            | datumVestigingInNederland | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
+
+
+
+      Abstract Scenario: <type> in <groep> <veld>
+        Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende '<e2e-tabel>' gegevens
+          | naam       | waarde     |
+          | <Lo3-naam> | <GbaDatum> |
+          Als personen wordt gezocht met de volgende parameters
+          | naam                | waarde                          |
+          | type                | RaadpleegMetBurgerservicenummer |
+          | burgerservicenummer | 000000152                       |
+          | fields              | <groep>.<veld>                  |
+          Dan heeft de response een persoon met de volgende '<groep>' gegevens
+          | naam               | waarde        |
+          | <veld>.type        | <type>        |
+          | <veld>.datum       | <datum>       |
+          | <veld>.jaar        | <jaar>        |
+          | <veld>.maand       | <maand>       |
+          | <veld>.onbekend    | <onbekend>    |
+          | <veld>.langFormaat | <langFormaat> |
+
+        Voorbeelden:
```

Comment:
Bij de gevallen waar in deze voorbeeldentabel de 1e kolom de waarde 'overlijden' heeft moet ook de property 'Indicatieoverleden' met de waarde 'true' gegenereerd worden.

Reply:
Thanks. Deze funtionaliteit (altijd indicatieoverleden leveren bij een overleden persoon) is nog niet goed in een feature terecht gekomen. Daar heb ik een issue voor op de sprintplanning gezet. 

Hier ook aanpassingen in gedaan om dit in de output af te dwingen.